library/build.gradle

```diff
@@ -2,6 +2,9 @@ subprojects {
 
   repositories {
     mavenCentral()
+//    maven {
```

Comment:
Nope, left it on purpose. Make it easier when one wants to test snapshot changes to HAHA. I never remember the damn line.